lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -619,7 +619,8 @@ E2Lib.OperatorChars = OperatorChars
 E2Lib.blocked_array_types = { -- todo: fix casing
 	["t"] = true,
 	["r"] = true,
-	["xgt"] = true
+	["xgt"] = true,
+	["f"] = true
```

Comment:
What? Why? Absolutely not.

Reply:
Okay, my bad, just thought they'd be messy.

Reply:
Only reason why arrays/tables/gtables aren't allowed is cause there's no way to absolutely ensure they are of that type. Functions have a metatable so you can be sure of it. Although I feel like the restriction could probably be lifted for tables if we end up implementing the metatable for that.

Reply:
Did not realize arrays were just naturally that bad at printing.

Reply:
> Did not realize arrays were just naturally that bad at printing.

They were worse before, I think if you tried to print an empty array it would print `[` or something. Much better refactored to use toString